homeUi/src/main/res/values/strings.xml

```diff
@@ -37,8 +37,14 @@
     <string name="gravatar_tab_more_options_make_current_content_description">Select avatar as current one</string>
     <string name="gravatar_tab_more_options_delete_avatar">Delete</string>
     <string name="gravatar_tab_more_options_delete_avatar_content_description">Delete this avatar</string>
+    <string name="gravatar_tab_more_options_download_avatar">Download Image</string>
+    <string name="gravatar_tab_more_options_download_avatar_content_description">Download Image</string>
     <string name="gravatar_tab_delete_confirmation_title">Delete Image</string>
     <string name="gravatar_tab_delete_confirmation_message">Are you sure you want to delete this image?</string>
     <string name="gravatar_tab_delete_confirmation_confirm">Delete</string>
     <string name="gravatar_tab_delete_confirmation_cancel">Cancel</string>
+    <string name="permission_required_alert_title">Permission required</string>
+    <string name="permission_required_rationale_message">To take a picture, you need to grant camera permission. You can grant it in the app settings.</string>
```

Comment:
We don't need this string.

Reply:
🤦  Thanks @AdamGrzybkowski 

It should be solved now.

<img width="409" alt="image" src="https://github.com/user-attachments/assets/6faaf6a8-1087-4cff-8554-f257fb291a04" />